crt_portal/cts_forms/tests/test_api.py

```diff
@@ -6,17 +6,17 @@
 from django.test import TestCase
 from django.test.client import Client
 from django.urls import reverse
-from ..models import Report
-from .test_data import SAMPLE_REPORT
+from ..models import Report, ResponseTemplate
+from .test_data import SAMPLE_REPORT, SAMPLE_RESPONSE_TEMPLATE
+from cts_forms.views import add_activity
 
 
 class APIBaseUrlTests(TestCase):
-
     def setUp(self):
         self.client = Client()
-        self.user = User.objects.create_user('DELETE_USER', 'george@thebeatles.com', '')
-        self.client.login(username='DELETE_USER', password='')  # nosec
-        self.url = reverse('api:api-base')
+        self.user = User.objects.create_user("DELETE_USER", "george@thebeatles.com", "")
```

Comment:
I was doing it because that is what is used throughout the program.  I assume they use that because that use should be deleted later, but I feel like its a little confusing.